handbook/handbook.people.onboarding.checklist.md

```diff
@@ -0,0 +1,256 @@
+---
+id: c6b422a5-204a-41dd-8e47-f0b736160790
+title: Checklist
+desc: ""
+updated: 1620248743423
+created: 1620248731246
+---
+
+## Welcome to Dendron!
+
+We are so excited you’re here and look forward to setting you up for [all-remote](https://about.gitlab.com/company/culture/all-remote/guide/) success as your journey begins. Copy this onboarding checklist and work through them over the course of your first 30 days. If any task doesn't make sense, please reach out to the team on discord. **Note:** We [lookup](https://handbook.dendron.so/notes/b89ba854-72fb-4ebc-a8a0-55960b89e9dc.html#lookup) to Gitlab for their focus on [single-source of truth](https://about.gitlab.com/handbook/values/#single-source-of-truth) handbook creation and would like to do the same for Dendron. We've included a lot of links to the Gitlab handbook and will tweak/subsume them into the Dendron handbook over time. The wealth of information in this note can be overwhelming at times. It's okay to skim through resources, bookmark them and come back at a later time.
```

Comment:
- change all-remote to remote-first (also rephrase latter part of sentence to go with this
- put instructions on where to copy to (eg. `user.{name}.onboarding`)
- don't feel great about *lookup to* phrase, instead of sprinkling it inline, can we have it at the top/bottom of the page?
```
## Lookups (link to value)
- this page is inspired and comprised of contents from the following sources: [Gitlab](link) (license), [Foo](link) (licesnse), etc

```

Reply:
This should be fixed now.